### DISCUSSION TITLE
Open Discussions

### DISCUSSION THREAD

**Discussion by hassankirwa** [Joymutheu-dev/MISSING-PERSONS-DATABASE-2024-KENYA-FINANCE-BILL-PROTESTS-#19] [General]
This is an open forum where we can have open discussions not related with issues.

**Comment by hassankirwa** [comment]
Hi Team ,
Hope you are well,
I am inviting all of you for this brief webinar we share some thoughts on contribution on the project above today at 2pm. Please respond if you will be available and if supposedly we dont have a quorum we reschedule.
MISSING-PERSONS-DATABASE-2024(discussion on github repo contribution)
Saturday, July 27 · 2:00 – 3:00pm
Time zone: Africa/Nairobi
Google Meet joining info
Video call link: https://meet.google.com/oyo-ourx-eog

Its a webinar , I am sharing it here so that incase i didnt capture your email .

**Comment by wathika-eng** [comment]
What's up with requirements.txt and Dockerfile?

Despite PR's being closed the files aren't updating on the main repo

https://github.com/Joymutheu-dev/MISSING-PERSONS-DATABASE-2024-KENYA-FINANCE-BILL-PROTESTS-/pull/18

https://github.com/Joymutheu-dev/MISSING-PERSONS-DATABASE-2024-KENYA-FINANCE-BILL-PROTESTS-/pull/14

**Reply by Jojo-devv** [reply]
@hassankirwa check
